### PR TITLE
Attempt using zeromq on windows

### PR DESCRIPTION
**Pull request by SylvainCorlay**

### CODE REVIEW

**Comment by SylvainCorlay** [comment]
Rahh, with `zeromq 4.2.5`, the `zeromq.lib` import libraries is not in `Library/lib` anymore but in `Library/bin`, which is **wrong**.

I am incredibly frustrated with zeromq, which seems to break in creative ways at every release.

**Comment by SylvainCorlay** [comment]
This appears to be working.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -21,14 +21,15 @@ requirements:
   build:
     - toolchain
     - pkg-config  # [not win]
+    - cmake  # [win]
```
Comment (minrk) [inline recipe/meta.yaml]: cmake can't be a depenency here, can it?

Comment (SylvainCorlay) [reply recipe/meta.yaml]: duh, right

Comment (SylvainCorlay) [reply recipe/meta.yaml]: Triggered a new build without it. Must have been working on two different things at the same time.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -23,12 +23,12 @@ requirements:
     - pkg-config  # [not win]
     - python
     - pip
-    - zeromq >=4.2.3,<4.3  # [not win]
-    - libsodium   # [not win]
+    - zeromq >=4.2.3,<4.3
```
Comment (minrk) [inline recipe/meta.yaml]: This is going to require zeromq 4.2.5 on Windows because of the .lib location fix, right?

Comment (SylvainCorlay) [reply recipe/meta.yaml]: Righy

**Comment by SylvainCorlay** [comment]
This is ready to go!

**Comment by minrk** [comment]
Awesome!

**Pull request merged**